_config/products.yml

```diff
@@ -140,9 +140,35 @@ devstudio:
   name: Red Hat CodeReady Studio
   url_path_fragment: devstudio
   streams:
-    simrel-2019-09:
-      12.x.Nightly:
+    simrel-2019-12:
+      12.14.x.Nightly:
         update_site_url: https://devstudio.redhat.com/12/snapshots/updates/
+    simrel-2019-09:
+      12.13.0.GA:
+        archived: false
+        update_site_url: https://devstudio.redhat.com/12/stable/updates/
+        marketplace_url: https://marketplace.eclipse.org/node/1616973
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2019-11-04
+        supported_devstudio_is_version: 12.12.0.GA
```

Comment:
@jeffmaury : not sure about the version here ..

Reply:
It's fine

Reply:
Should be 12.11.0.GA